src/blog/_posts/2017-06-22-opera-46.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What’s new in Chromium 59 and Opera 46
+authors:
+- simon-pieters
+intro: 'Opera 46 (based on Chromium 59) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To find out what’s new for users, see our [Desktop](https://www.opera.com/blogs/desktop/) blog. Here’s what it means for web developers.'
+tags:
+- html
+- css
+- dom
+- javascript
+- opera
+license: cc-by-3.0
+---
+
+Opera 46 (based on Chromium 59) for [Mac, Windows,
+Linux](https://www.opera.com/computer) is out! To find out what's new
+for users, see our [Desktop](https://www.opera.com/blogs/desktop/)
+blog. Here's what it means for web developers.
+
+## Animated PNG
+
+Opera now supports animated PNG, or APNG for short. APNG is a file format that works similarly to
+GIF. The difference is that APNG is smaller and supports both 24-bit images and 8-bit transparency.
+It has become quite popular recently, particularly since Apple [adopted the APNG file format for iOS
```

Comment:
"...for the iOS..."

(And should the link perhaps only cover "the iOS 10 iMessage apps"? Just a thought.)